benches/mkbench.rs

```diff
@@ -54,11 +58,16 @@ type BenchFn<R> = fn(
     ReplicaToken,
     &Arc<Log<'static, <<R as ReplicaTrait>::D as Dispatch>::WriteOperation>>,
     &Arc<R>,
-    &Operation<
-        <<R as ReplicaTrait>::D as Dispatch>::ReadOperation,
-        <<R as ReplicaTrait>::D as Dispatch>::WriteOperation,
+    &Vec<
+        Operation<
+            <<R as ReplicaTrait>::D as Dispatch>::ReadOperation,
+            <<R as ReplicaTrait>::D as Dispatch>::WriteOperation,
+        >,
     >,
-    usize,
+    usize, // Operating vector length
```

Comment:
Not 100% sure but I think we can name parameters like this: type myfun = fn(a: usize, b: usize),

we should probably give names now that the function is so huge?